### PR TITLE
Add Karma package

### PR DESCRIPTION
**Pull request by sbruzzese902**
As per the PR title, I'm adding Karma as part of the monitoring module.

There aren't official Karma manifests for Kubernetes yet, so this is a simple standard setup.

Any comment or suggestion is more than welcomed 😄

### CODE REVIEW

File: katalog/karma/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- Karma image: `registry.sighup.io/fury/karma/karma:v0.113`
+- Karma repository: [Karma on GitHub][k-gh]
+
+## Configuration
+
+Fury distribution Karma is deployed with the following
+configuration:
+- Alertmanager URI: `http://alertmanager-main.monitoring.svc.cluster.local:9093`
+- Polling interval: `1m`
```
Comment (ralgozino) [inline katalog/karma/README.md:28]: linter will complain if you don't leave an empty line before a list
```suggestion
configuration:

- Alertmanager URI: `http://alertmanager-main.monitoring.svc.cluster.local:9093`
- Polling interval: `1m`
```

File: katalog/karma/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+All configuration options can be found [here](https://github.com/prymitive/karma/blob/main/docs/CONFIGURATION.md)
```
Comment (ralgozino) [inline katalog/karma/README.md:30]: I suggest linking to the docs for the version we are using:

```suggestion
All configuration options can be found [here](https://github.com/prymitive/karma/blob/v0.113/docs/CONFIGURATION.md)
```

pointing to `main` has the downside that the file contents could change or even disappear :)

File: katalog/karma/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Deployment
+You can deploy karma by running the following command:
```
Comment (ralgozino) [inline katalog/karma/README.md:33]: linting
```suggestion
## Deployment

You can deploy karma by running the following command:
```

maybe we can say something about how to create the configuration patch here?

File: katalog/karma/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            path: /health
+            port: 8080
+        volumeMounts:
+        - mountPath: /etc/karma
+          name: config
+          readOnly: false
+      volumes:
+      - configMap:
+          name: karma-config
+        name: config
```
Comment (ralgozino) [inline katalog/karma/deployment.yaml:53]: nitpick: missing new line at the end of the file

**Review by ralgozino — CHANGES_REQUESTED** [review]
some minor corrections

**Review by ralgozino — CHANGES_REQUESTED** [review]

**Review by ralgozino — CHANGES_REQUESTED** [review]
Sorry Simo for the several requests for change, I just realised that the main readme should be updated to add Karma to the packages table.

The rest lgtm

**Comment by sbruzzese902** [comment]
No problem! That's the reason for reviews 😄 I'm fixing it!

**Review by ralgozino — APPROVED** [review]
LGTM 💪

**Review by nutellinoit — APPROVED** [review]
LGTM 🚀

**Pull request merged**